test/iterator-test.js

```diff
@@ -3,31 +3,38 @@ const test       = require('tape')
     , leveldown  = require('../')
     , abstract   = require('abstract-leveldown/abstract/iterator-test')
     , make       = require('./make')
+    , iota       = require('iota-array')
+    , lexi       = require('lexicographic-integer')
+    , util       = require('util')
 
 abstract.all(leveldown, test, testCommon)
 
-make('iterator throws if key is not a string', function (db, t, done) {
-  var ite = db.iterator()
-  var error
-  try {
-    ite.seek()
-  } catch (e) {
-    error = e
-  }
+make('iterator seek throws if key is not a string or buffer', function (db, t, done) {
+  var keys = [void 0, 10, true, null]
```

Comment:
is there a reason not to use `undefined` instead of `void 0`? void is quite uncommon these days